### DISCUSSION TITLE
Some services are not registered with SkyWalking.

### DISCUSSION THREAD

**Discussion by linshenfanying** [apache/skywalking#12327] [Q&A]
My Java services are all deployed in Kubernetes, and SkyWalking is also deployed in Kubernetes. However, some of my services can be found with their Service names on the SkyWalking page, but many cannot. All the Java services are using the same agent configuration.

**Comment by wu-sheng** [comment]
Your description doesn't make any sense, I am afraid. There is no mechanism to target specific services.

**Reply by linshenfanying** [reply]
In the namespace Test, I have two services named A and B. Both services use the same version of Java agent, with the same configuration file and the same startup method. However, service A can be found in the UI, but service B cannot.
